<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1050</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="Lbl_Question">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>50</y>
     <width>600</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard ExtraBold</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>햄버거를 골라주세요.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_back">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>300</width>
     <height>130</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard Variable</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>뒤로가기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_next">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1610</x>
     <y>40</y>
     <width>300</width>
     <height>130</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard Variable</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>다음 단계로</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_bulgogiBugerImg">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/buger/bulgogi.png</normaloff>../img/buger/bulgogi.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>400</width>
     <height>300</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_cheeseBugerImg">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>280</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/buger/cheese.png</normaloff>../img/buger/cheese.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>400</width>
     <height>400</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_chickenBugerImg">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>280</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/buger/chicken.png</normaloff>../img/buger/chicken.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>400</width>
     <height>400</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>540</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard Variable</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>불고기 버거</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>540</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard Variable</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>치즈 버거</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>540</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard Variable</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>치킨 버거</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>540</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard Variable</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>필레 오 피쉬</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_filetofishBugerImg">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>280</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/buger/filetofish.png</normaloff>../img/buger/filetofish.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>400</width>
     <height>400</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_bigmacBugerImg">
   <property name="geometry">
    <rect>
     <x>1490</x>
     <y>280</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/buger/bigmac.png</normaloff>../img/buger/bigmac.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>400</width>
     <height>400</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>540</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard Variable</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>더블 필레오 피쉬</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1520</x>
     <y>540</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard Variable</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>빅맥</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_doublefiletofishBugerImg">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>280</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/buger/doublefiletofish.png</normaloff>../img/buger/doublefiletofish.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>400</width>
     <height>400</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl_bulgogi">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard Variable</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl_cheese">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>270</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard Variable</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl_chicken">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>270</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard Variable</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl_filetofish">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>270</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard Variable</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl_bigmac">
   <property name="geometry">
    <rect>
     <x>1710</x>
     <y>270</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard Variable</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl_doublefiletofish">
   <property name="geometry">
    <rect>
     <x>1440</x>
     <y>270</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard Variable</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
